DEBUG Capture output from timeout

jugglinmike opened the pull request:
This branch is being used as part of an investigation on intermittent
failures in the continuous integration environment. It is not intended
to be merged to `master`.

<!-- Reviewable:start -->
---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/w3c/web-platform-tests/5544)
<!-- Reviewable:end -->

jugglinmike committed:
DEBUG Capture output from timeout

This branch is being used as part of an investigation on intermittent
failures in the continuous integration environment. It is not intended
to be merged to `master`.

w3c-bots commented:
These tests are now available on [w3c-test.org](https://w3c-test.org/submissions/5544/)

jugglinmike committed:
Output logging information directly

The previous commit demonstrated that TravisCI cancels all lifecycle
events following a job timeout. Because of this, debugging information
cannot be buffered and emitted in the event of a timeout--it must be
output in real time.

Update the build script to output debugging information in real time.
Remove command intended to force a job timeout.

jugglinmike committed:
Trigger build on TravisCI

jugglinmike commented:
The required information has been captured.
